db/test/test_dataset.py

```diff
@@ -169,6 +169,61 @@ def test_delete_shapshot(self):
         with self.assertRaises(db.exceptions.NoDataFoundException):
             dataset.get_snapshot(snap_id)
 
+    def test_add_recordings(self):
+        id = dataset.create_from_dict(self.test_data, author_id=self.test_user_id)
+        test_data_recordings = {
+            "class_name": "Class #1",
+            "recordings": ["1c085555-3805-428a-982f-e14e0a2b18e6"]
+        }
+        originaldataset = dataset.get(id)
+        dataset.add_recordings(test_data_recordings, id)
+        updateddataset = dataset.get(id)
+        self.assertDictEqual(originaldataset["classes"][1], updateddataset["classes"][1])
+        self.assertEqual(len(originaldataset["classes"][0]["recordings"])+1, len(updateddataset["classes"][0]["recordings"]))
+        for mbid in updateddataset["classes"][0]["recordings"]:
+            if mbid not in originaldataset["classes"][0]["recordings"]:
+                self.assertEqual(mbid, test_data_recordings["recordings"][0])
+
+    def test_delete_recordings(self):
+        id = dataset.create_from_dict(self.test_data, author_id=self.test_user_id)
+        test_data_recordings = {
+            "class_name": "Class #1",
+            "recordings": ["ed94c67d-bea8-4741-a3a6-593f20a22eb6"]
+        }
+        originaldataset = dataset.get(id)
+        dataset.add_recordings(test_data_recordings, id)
+        dataset.delete_recordings(test_data_recordings, id)
+        updateddataset = dataset.get(id)
```

Comment:
Here, you call `add_recordings` and then `delete_recordings` directly one after another without checking the result. This test passes, but it would also pass if the add and delete methods didn't do anything.
A better test here would be to delete a single recording which already exists in `self.test_data` and then do a similar test to what I proposed in `test_add_recordings` to check that the list of recordings is smaller than the original dataset list.